Tests/InstrumentationTests/URLSessionTests/URLSessionInstrumentationTests.swift

```diff
@@ -250,7 +260,7 @@ class URLSessionInstrumentationTests: XCTestCase {
         XCTAssertTrue(URLSessionInstrumentationTests.checker.receivedErrorCalled)
     }
 
-    public func testShouldInstrumentRequest() throws {
+    public func testShouldInstrumentRequest_PropagateCombinedActiveAndDefaultBaggages() throws {
```

Comment:
Could this functionally be tested in different tests? Just move baggage evaluation to two new different tests, and keep the existing test without baggage evaluation

Reply:
Makes sense! Baggage logic is now in separate tests, and the original test stays focused on the basics.